services/pdf-ms/serverless.yml

```diff
@@ -44,6 +44,21 @@ provider:
         - dynamodb:UpdateItem
       Resource:
         - Fn::ImportValue: ${self:custom.resourcesStage}-CasesTableArn
+    - Effect: Allow
+      Action:
+        - dynamodb:Query
+      Resource:
+        - Fn::Join:
+          - ""
+          - - Fn::ImportValue: ${self:custom.resourcesStage}-CasesTableArn
+            - "/index/*"
+    - Effect: Allow
+      Action:
+        - dynamodb:Query
+        - dynamodb:Scan
```

Comment:
You get the hole document with scan. is this what you want?

Reply:
I copied this from the usersApi (which I think you wrote 😃), where we allow both Query, Scan and getItem; so I'm unsure. Since I'm using dynamoDb.call('get', params) is getItem all I need? Sounds reasonable, I'll test it, and if so, we should also change this in the usersApi.

Reply:
Checked this, and I only need getItem, so removing the other two. Same change can be made in the usersApi.